src/main/java/appeng/parts/networking/PartQuartzFiber.java

```diff
@@ -179,27 +212,26 @@ public double injectAEPower( final double amt, final Actionable mode, final Set<
 		{
 			final IEnergyGrid eg = this.outerProxy.getEnergy();
 			if( !seen.contains( eg ) )
-			{
-				return eg.injectAEPower( amt, mode, seen );
-			}
+				amount = eg.injectProviderPower( amount, mode, seen );
 		}
 		catch( final GridAccessException e )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1166)